backend/tournesol/migrations/0033_alter_entity_tournesol_score.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2022-02-22 13:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tournesol', '0032_entity_tournesol_score'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='entity',
+            name='tournesol_score',
+            field=models.FloatField(default=0, help_text='Tournesol ScoreCalculated from the criteria_score from certified contributors'),
+        ),
+    ]
```

Comment:
You have created to much migrations for this issue. You can combine them into one squash.
The more migrations you create, the longer the database will take to update its columns